sandbox/registration-processor-dmz.properties

```diff
@@ -159,6 +159,11 @@ mosip.registration.processor.sync.version=1.0
 mosip.registration.processor.registration.status.version=1.0
 mosip.registration.processor.lostrid.version=1.0
 
+#Registration External Status Id
+mosip.registration.processor.registration.external.status.id=mosip.registration.external.status
```

Comment:
add this line after the mosip.registration.processor.lostrid.id  and the version after the mosip.registration.processor.lostrid.version with the comments. Please use space after the #

Reply:
done.

Reply:
ok resolved